### PR TITLE
Adding details for running Docker Container

### PR DESCRIPTION
**Pull request by solankimihir**
Added docker command for running Docker Container with prebuilt containers

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -100,6 +100,24 @@ npm run prod
 
 _Although the build of the client is production-ready, Nuxt's own HTTP server should never be exposed directly to the internet and a proper hosting tool like Nginx should always be used instead._
 
+</br>
+
+## Running Docker Container
+
+---
```
Comment (ferferga) [inline README.md:107]: No needed as the header already adds a divisor

**Review by ferferga — CHANGES_REQUESTED** [review]
Please, also follow the conventional commits guidelines https://www.conventionalcommits.org/en/v1.0.0/

File: README.md [hunk 2]
```diff
@@ -100,6 +100,24 @@ npm run prod
...
+
+```bash
+docker run -d -p 8080:80 ghcr.io/jellyfin/jellyfin-vue
+```
+
+You can select replace port "8080" with your choice of port.
+
+You can refer [Packages](https://github.com/jellyfin/jellyfin-vue/pkgs/container/jellyfin-vue) for tagged image versions.
+
+</br>
```
[the remark this answers is not in the document]
Comment (ThibaultNocchi) [reply README.md:119]: I'd even leave out the `-d`, keeping the bare minimum. People can then add whatever they need.

File: README.md (hunk 1 above)
Comment (solankimihir) [reply README.md:107]: Noted.

File: README.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (solankimihir) [reply README.md:119]: Leaving out "-d" flag may cause confusion among users who are experimenting with docker for first time. 
If we decide to leave out "-d" flag to keep it at bare minimum, can we consider adding a note below the command explaining that "-d" flag can be added to run the docker-container in backgroud?

Side note: On [Jellyfin documentation for Docker](https://jellyfin.org/docs/general/administration/installing/#docker) "-d" flag is present in their docker run command.

**Comment by solankimihir** [comment]
Review points closed. However, "-d" flag is kept in the docker-run command. Consider merging the PR.

**Pull request merged**